views/about-page.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="/public/styles.css" rel="stylesheet" type="text/css" />
+    <title>about</title>
+  </head>
+  <body>
+    <nav>
+      <div class="navbar-nav">
+        <a class="nav-item nav-link active" href="#">Home </a>
+        <a class="nav-item nav-link" href="#">About</a>
+        <a class="nav-item nav-link" href="#">Works</a>
+      </div>
+    </nav>
+    <div class="container">
+      <div>
+        <img src="/public/images/inigo-kennedy-port.jpg" alt="inigo-kennedy" />
```

Comment:
again you don't need the /public/ here, just the path after it
`<img src="/images/inigo-kennedy-port.jpg" alt="inigo-kennedy" />`